<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Res.qrc">
    <normaloff>:/images/AppIcon.ico</normaloff>:/images/AppIcon.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>36</width>
    <height>36</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="separator"/>
    <addaction name="actClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
    <addaction name="actCut"/>
    <addaction name="separator"/>
    <addaction name="actClear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actFontBold"/>
    <addaction name="actFontItalic"/>
    <addaction name="actFontUnder"/>
    <addaction name="separator"/>
    <addaction name="actToolbarLab"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actOpen"/>
   <addaction name="actClose"/>
   <addaction name="separator"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="actCut"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actFontBold"/>
   <addaction name="actFontItalic"/>
   <addaction name="actFontUnder"/>
   <addaction name="separator"/>
  </widget>
  <action name="actNew">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/new2.bmp</normaloff>:/images/images/new2.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/open3.bmp</normaloff>:/images/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/103.bmp</normaloff>:/images/images/103.bmp</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空文本</string>
   </property>
  </action>
  <action name="actClose">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/exit.bmp</normaloff>:/images/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actCut">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/cut.bmp</normaloff>:/images/images/cut.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/120.bmp</normaloff>:/images/images/120.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/paste.bmp</normaloff>:/images/images/paste.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actFontBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/BLD.BMP</normaloff>:/images/images/BLD.BMP</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actFontItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/ITL.BMP</normaloff>:/images/images/ITL.BMP</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actFontUnder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/UNDRLN.BMP</normaloff>:/images/images/UNDRLN.BMP</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actFont">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/images/20.ico</normaloff>:/images/images/20.ico</iconset>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
   <property name="toolTip">
    <string>字体</string>
   </property>
  </action>
  <action name="actToolbarLab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示工具栏文字标签</string>
   </property>
   <property name="toolTip">
    <string>显示工具栏文字标签</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCopy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClear</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actPaste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
